### PR TITLE
Document usage in the desktop app

### PR DESCRIPTION
**Pull request by johansatge**
This repository made my day, thank you ❤️ 

Here is a small doc update to support the desktop app too.

(Unfortunately I don't know how to enable developer mode on Windows, feel free to edit 😃)

Cheers,

### CODE REVIEW

**Comment by steppefox** [comment]
On Mac it's easy to hide it, just press "Aa" button:

image

**Comment by johansatge** [comment]
Unfortunately @steppefox the toggle button hides the UI, but the feature stays active (wysiwyg as you type your messages)

File: README.md [hunk 1]
```diff
@@ -14,8 +14,20 @@ functionality, you can use this bookmarklet to temporarily disable all
...
 1. Copy the contents of `index.js`
 2. Paste into the URL field of a browser bookmark
 3. Navigate to a Slack workspace in that browser
 4. Execute the bookmarklet to disable the WYSIWYG editor
 5. Repeat each time you reload slack
+
+### In the desktop app
+
+1. Open the slack app in Developer mode
+  * On Mac: `export SLACK_DEVELOPER_MENU=true && open -a /Applications/Slack.app`
```
Comment (adamralph) [inline README.md:29]:
```suggestion
  * On Mac: `export SLACK_DEVELOPER_MENU=true && open -a /Applications/Slack.app`
  * On other OS's: ensure the `SLACK_DEVELOPER_MENU` environment variable is set to `true` before opening the Slack app.
```

File: README.md [hunk 2]
```diff
@@ -14,8 +14,20 @@ functionality, you can use this bookmarklet to temporarily disable all
...
+2. Right-click in the app and _Inspect element_
+3. Copy the contents of `index.js`
+4. Paste into the Console
```
Comment (adamralph) [inline README.md:32]:
```suggestion
4. Paste into the Console tab
```

File: README.md [hunk 3]
```diff
@@ -14,8 +14,20 @@ functionality, you can use this bookmarklet to temporarily disable all
...
+5. Repeat each time you reload slack
```
Comment (adamralph) [inline README.md:33]:
```suggestion
5. Repeat each time you reload slack or switch to a team for the first time
```

**Review by adamralph — CHANGES_REQUESTED** [review]

**Comment by MitMaro** [comment]
Would it be worth adding that the environment variable can be added to your shell profile to make that step persistent?

**Comment by adamralph** [comment]
I don't know that we need to go into all the various of methods of setting environment variables in all OS's.

**Comment by MitMaro** [comment]
Oh I agree with that! There are too many different setups.

I meant just making a single note stating the environment variable can be set in your shell profile to avoid having to run the command from your terminal. For me, on Linux, once it is set I can use the default launcher without needing to run the command from the command line.

**Comment by lietu** [comment]
On Windows it's `View Advanced system settings` -> `Environment Variables...` -> `New...` (User or System)

Alternatively in Administrator PowerShell:
```powershell
[System.Environment]::SetEnvironmentVariable("SLACK_DEVELOPER_MENU", "true", "User")
# OR
[System.Environment]::SetEnvironmentVariable("SLACK_DEVELOPER_MENU", "true", "Machine")
```

Or in cmd.exe (again, as Administrator):
```batch
reg add HKEY_CURRENT_USER\Environment /V SLACK_DEVELOPER_MENU /t REG_SZ /d true
rem OR
reg add HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment /V SLACK_DEVELOPER_MENU /t REG_SZ /d true
```

**Comment by kfahy** [comment]
> Here is a small doc update to support the desktop app too.

Brilliant! Thanks a lot for helping out desktop app users too.

And thanks to everyone for the feedback on this functionality. I wrote this hack for fun last night and didn't think anyone would find it useful, but it looks like a lot of us are dissatisifed with the new editor. I hope that Slack takes note of this dissatisfaction, and soon builds a preference to revert back to the old editor. Then, we can delete this repository and live happily ever after.

Until then, if anyone knows of a better approach than removing experiments via redux (also, what is up with their action type naming?), feel free to submit a PR.

File: README.md (hunk 1 above)
Comment (kfahy) [reply README.md:29]: Thanks for the amendments! Doesn't look like I have permissions to push to this branch, so I'm going to merge as-is, and we can follow up with these changes after.

**Pull request merged**